packages/shared/src/components/video/VideoPlayOverlay.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react';
+import PlayIcon from '../icons/Play';
+import { IconSize } from '../Icon';
+
+interface VideoPlayOverlayProps {
+  size?: IconSize;
+}
+
+const VideoPlayOverlay = ({
```

Comment:
After updating the `Image` component, we can probably turn this to just a `classed` component so its children will be centered `PlayIcon`. Feels like there's a good chance for things to break when you don't control the container of the absolute elements. Just not really a fan of absolute elements without the container defined side-by-side as it is less predictable. Probably just a personal preference, but would love to hear other people's thoughts. cc @dailydotdev/web-team